forge-gui/res/cardsfolder/upcoming/fishers_talent.txt

```diff
@@ -0,0 +1,15 @@
+Name:Fisher's Talent
+ManaCost:2 G U
+Types:Enchantment Class
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPeek | TriggerDescription$ At the beginning of your upkeep, look at the top card of your library. You may reveal it if it's a land card. Create a 1/1 blue Fish creature token if you revealed it this way. Then draw a card.
+SVar:TrigPeek:DB$ PeekAndReveal | PeekAmount$ 1 | RevealValid$ Land | RevealOptional$ True | RememberRevealed$ True | SubAbility$ DBToken
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | ConditionDefined$ Remembered | ConditionPresent$ Card | ConditionCompare$ GE1 | TokenScript$ u_1_1_fish | TokenOwner$ You | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+K:Class:2:G U:AddReplacementEffects$ SharkCatch
+SVar:SharkCatch:Event$ CreateToken | ActiveZones$ Battlefield | ValidPlayer$ You | ValidToken$ Fish | ReplaceWith$ TokenReplace1 | Description$ If you would create a Fish token, create a 3/3 blue Shark creature token instead.
+SVar:TokenReplace1:DB$ ReplaceToken | Type$ ReplaceToken | ValidCard$ Fish | TokenScript$ u_3_3_shark
+K:Class:3:2 G U:AddReplacementEffects$ OctopusCatch
+SVar:OctopusCatch:Event$ CreateToken | ActiveZones$ Battlefield | ValidPlayer$ You | ValidToken$ Shark | ReplaceWith$ TokenReplace2 | Description$ If you would create a Shark token, create an 8/8 blue Octopus creature token instead.

```

Comment:
About the SVars, can you check if the Code doesn't break when using cards like `Artificial Evolution` ?

So for example, when you alter `Shark` into `Human` for example, would the Fish token create a Human token instead?

Reply:
Nope, that doesn't work.

Reply:
@tool4ever we might look into this because it is related to the Class Keyword :/ (and how Keywords are done with card text changes)

Reply:
Referenced in new Issue: #5649